app/src/main/res/layout/row_item.xml

```diff
@@ -9,7 +9,7 @@
     android:padding="@dimen/album_art_border">
 
     <ImageView
-        android:id="@+id/album_art"
+        android:id="@+id/imageView_albumArt"
```

Comment:
You are mixing naming conventions here - resources are generally named with underscores between words (except for android view types where we keep as one word) rather than camel case. So I would name this `imageview_album_art`